docs/en/ingest-management/elastic-agent/configuration/outputs/output-elasticsearch.asciidoc

```diff
@@ -0,0 +1,586 @@
+:output-type: elasticsearch
+
+[[elasticsearch-output]]
+== {es} output
+
+The `elasticsearch` output sends events directly to {es} by using the
+{es} HTTP API.
+
+*Compatibility:* This output works with all compatible versions of {es}. See the
+https://www.elastic.co/support/matrix#matrix_compatibility[Elastic Support
+Matrix].
+
+Example configuration:
+
+//TODO: Provide a example that shows more of the settings users are likely to
+//change.
+
+[source,yaml]
+----
+output.elasticsearch:
+  hosts: ["https://myEShost:9200"] <1>
+----
+<1> To enable SSL, add `https` to all URLs defined under __hosts__.
+
+The `elasticsearch` output supports the following settings, grouped by category.
+Many of these settings have sensible defaults that allow you to run {agent} with
+minimal configuration.
+
+<<output-elasticsearch-commonly-used-settings>>::
+Settings frequently changed for this input type.
+
+<<output-elasticsearch-authentication-settings>>::
+Settings for authenticating with {es}.
+
+<<output-elasticsearch-data-parsing-settings>>::
+Settings used to parse, filter, and transform data.
+
+<<output-elasticsearch-http-settings>>::
+Settings that modify the HTTP request sent to {es}.
+
+<<output-elasticsearch-performance-tuning-settings>>::
+Settings that control timeouts and other behavior that affects performance.
+
+
+//QUESTION: Do we still support all the ways to authenticate against ES, or
```

Comment:
I could use more context here to help push the conversation on.  Maybe pull a list of questions to an email thread (or git ticket) and we can knock them down.

Reply:
question deleted...I was misunderstanding how things work.